CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.37.0] - 2022-06-23
+- Translate `:restli-common` PDL files to PDSC files to support legacy web use cases.
```

Comment:
This is wrong, it's not translating anything. I'd say "Package translate legacy PDSC models into :restli-common JAR"

Also remove LI-specific comment (i.e. "web")